Makefile-decls.am

```diff
@@ -43,7 +43,7 @@ gir_DATA =
 typelibdir = $(libdir)/girepository-1.0
 typelib_DATA =
 gsettings_SCHEMAS =
-ostree_bootdir = /usr/lib/ostree
+ostree_bootdir = $(prefix)/usr/lib/ostree
```

Comment:
This is wrong actually:

http://build.gnome.org/continuous/buildmaster/builds/2016/08/11/29/build/log-ostree.txt
```
  /bin/sh ./libtool   --mode=install /usr/bin/install -c ostree-prepare-root ostree-remount '/ostbuild/results/usr/usr/lib/ostree'
```